app/models/workers_scaling/experiment_statistics_factory.rb

```diff
@@ -0,0 +1,19 @@
+module WorkersScaling
+  class ExperimentStatisticsFactory
+    @cache = ActiveSupport::Cache::MemoryStore.new expires_in: 10.minutes
```

Comment:
Tak, ustawia to dla wszystkich wpisów. 
MemoryStore - lokalny cache w tym samym procesie
MemCacheStore - centralny serwer z cashem który trzeba dodatkowo konfigurować, jeśli stworzymy bez konfiguracji działa lokalnie co pozbawia sensu używania czegoś takiego.
http://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-store
Jesteśmy przygotowani na używanie centralnego cashe w Scalarmie?